crates/uv-requirements/src/specification.rs

```diff
@@ -564,17 +279,274 @@ impl RequirementsSpecification {
     pub async fn from_simple_sources(
         requirements: &[RequirementsSource],
         client_builder: &BaseClientBuilder<'_>,
-        preview: PreviewMode,
     ) -> Result<Self> {
-        Self::from_sources(
-            requirements,
-            &[],
-            &[],
-            None,
-            &ExtrasSpecification::None,
-            client_builder,
-            preview,
-        )
-        .await
+        Self::from_sources(requirements, &[], &[], client_builder).await
+    }
+}
+
+// TODO(konsti): Hook this into source tree handling.
+/*
+#[cfg(test)]
```

Comment:
These were intended as unit tests for the lowering, but the api that i used to the reading went away so now we need to hook this somewhere different